test/it/client-listUsers.js

```diff
@@ -25,7 +41,7 @@ describe('client.listUsers()', () => {
   });
```

Comment:
In record mode, the ID for the user will be stored in the tape.
In playback mode, when you create a user, you will get back the same user ID that was used, when the user was created in playback mode. (ID would be returned by the real server during playback mode)
So, you wouldn't really include an arbitrary ID in the request. You'd use the ID that was returned to your by the response in a tape. 
@lboyette-okta - Does this sound right?